### PR TITLE
MC-18820: Delete Route & Internet Gateway API docs

### PR DESCRIPTION
**Pull request by joelplourde4**
### Fixes [MC-18820](https://cloud-ops.atlassian.net/browse/MC-18820)

### CODE REVIEW

**Review by Vioarr — APPROVED** [review]

File: source/includes/aws/_routes.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+The destination for the route is represented by a IPV4 CIDR block (e.g: 0.0.0.0/0, which represents all IPv4 addresses).
+The target is the internet gateway that's attached to your VPC.
+
+<!-------------------- DELETE ROUTES -------------------->
+
+#### Delete a Route
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/aws/aws-tesv/routes/0.0.0.0/0?operation=delete?routeTableId=rtb-04703a509a773f41b"
```
Comment (emilyroser) [inline source/includes/aws/_routes.md]:
```suggestion
   "https://cloudmc_endpoint/v1/services/aws/aws-tesv/routes/0.0.0.0/0?operation=delete&routeTableId=rtb-04703a509a773f41b"
```

File: source/includes/aws/_internetgateways.md [hunk 2]
```diff
@@ -214,4 +214,39 @@ Detach an internet gateway from a VPC in a given [environment](#administration-e
...
\ No newline at end of file
+| `deleteRoutes`<br/>*boolean*                     | Whether or not to delete the routes attached to the Internet Gateway. Default to true. |
+
+<!-------------------- DELETE AN INTERNET GATEWAY -------------------->
+
+#### Delete an Internet Gateway
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/aws/aws-tesv/internetgateways/igw-00b5e5bb6cce152cb?operation=delete"
```
Comment (emilyroser) [inline source/includes/aws/_internetgateways.md:225]: i'm getting an` internetGateway 'igw-0d725a79ba2de538c' has dependencies and cannot be deleted` error in my console when I try this 👀  is that expected under some conditions?

**Review by emilyroser — CHANGES_REQUESTED** [review]
small comments!! :))

File: source/includes/aws/_internetgateways.md (hunk 2 above)
Comment (joelplourde4) [reply source/includes/aws/_internetgateways.md:225]: Good question, I never encounter that issue before, will need to look into it.

Comment (joelplourde4) [reply source/includes/aws/_internetgateways.md:225]: Was it in dev? Can you link me the request itself?

Comment (emilyroser) [reply source/includes/aws/_internetgateways.md:225]: it was on my local, i'll try finding one in dev

Comment (emilyroser) [reply source/includes/aws/_internetgateways.md:225]: here's one on dev that errors out `https://portal.dev.cloudmc.io/api/v1/services/aws-aaaa/cloudmc-aws-dev/internetgateways/igw-0389447eadb99ba3d?operation=delete`

Comment (joelplourde4) [reply source/includes/aws/_internetgateways.md:225]: I just tried it and it worked on my side damnit.

Comment (emilyroser) [reply source/includes/aws/_internetgateways.md:225]: if you go in the UI it's deleted & audit event is there?

Comment (joelplourde4) [reply source/includes/aws/_internetgateways.md:225]: oh yeah I am mistaken, it worked but the task failed!

Comment (joelplourde4) [reply source/includes/aws/_internetgateways.md:225]: Created a JIRA ticket for it: https://cloudops.atlassian.net/browse/MC-18280

Comment (joelplourde4) [reply source/includes/aws/_internetgateways.md:225]: Here's a PR: https://github.com/cloudops/cloudmc-aws-plugin/pull/291

Comment (emilyroser) [reply source/includes/aws/_internetgateways.md:225]: awesome!! lgtm then :))

**Review by emilyroser — APPROVED** [review]

**Pull request merged**